README.md

```diff
@@ -21,27 +21,43 @@ Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter.
 This code requires the glmnet_matlab package (http://web.stanford.edu/~hastie/glmnet_matlab/download.html).
 Unzip `glmnet_matlab.zip` in either the root directory (that contains `SCINGE_Example.m`) or the `code` subdirectory.
 
+
+## Modes of Execution
+SINGE can be executed in the following ways:
+
+**MATLAB environment**
+```
+SINGE(Data,gene_list,outdir,hyperparameter_file)
+```
+**Compiled MATLAB code with R2018a runtime**
+```
+sh standalone_SINGE.sh Data gene_list outdir hyperparameter_file runtime_dir
```

Comment:
`bash` instead of `sh` to match the first line of the script?

We can also give an example with the actual data files used during testing.